lib/fast_inserter/version.rb

```diff
@@ -1,3 +1,3 @@
 module FastInserter
-  VERSION = "0.1.6"
+  VERSION = "0.1.7"
```

Comment:
after we verify that this PR works well in our production codebase, we should consider moving to `1.0` since it is production ready code for some time now